test/Dommel.Tests/SqlExpressions/SqlExpressionTests.cs

```diff
@@ -12,5 +12,14 @@ public void ToString_ReturnsSql()
             Assert.Equal(" where ([Name] = @p1)", sql);
             Assert.Equal(sql, _sqlExpression.ToString());
         }
+
+        [Fact]
+        public void ToStringVisitation_ReturnsSql()
+        {
+            var _sqlExpression = new SqlExpression<Product>(new SqlServerSqlBuilder());
```

Comment:
Local variables should not be prefixed. Rename `_sqlExpression` to `sqlExpression`.

Reply:
Took the liberty to apply that to your prev. commit 😄